packages/ember-simple-auth-oauth2/lib/simple-auth-oauth2/authenticators/oauth2.js

```diff
@@ -218,13 +231,15 @@ export default Base.extend({
     @protected
   */
   makeRequest: function(url, data) {
-    return Ember.$.ajax({
-      url:         url,
-      type:        'POST',
-      data:        data,
-      dataType:    'json',
-      contentType: 'application/x-www-form-urlencoded'
-    });
+    var _this = this;
+    var options = {url: url, type: 'POST', data: data, dataType: 'json', contentType: 'application/x-www-form-urlencoded'};
+
+    if(!Ember.isEmpty(_this.clientId)) {
+      var base64ClientId = btoa(_this.clientId + ":");
```

Comment:
Is vendor the correct place to put this, and add it to the list of files grunt grabs?  When I did this it appears to add the polyfill to every package, when in reality it is only needed for the OAuth2 authentication right now.

* I added base64.js to /vendor
* Updated grunt concat config to add vendor/base64.js

```
files['tmp/' + pkg.name + '.js'] = ['packages/' + pkg.name + '/wrap/browser.start', 'packages/' + pkg.name + '/wrap/register-library', 'vendor/loader.js', 'vendor/base64.js', 'tmp/libs/' + pkg.main + '.js', 'tmp/libs/' + pkg.main + '/**/*.js', 'packages/' + pkg.name + '/wrap/browser.end'];
```